wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/command/CommandDecoratorTest.java

```diff
@@ -182,4 +182,27 @@ void perpetuallyTest() {
       assertTrue(scheduler.isScheduled(perpetual));
     }
   }
+
+  @Test
+  void unlessTest() {
```

Comment:
maybe split this test into two? I'm a bit concerned about leftover state.

using parameterized tests, you can just have the same test, just once for `true` and once for `false`.
not sure how you'd do this in C++ though